master.tf

```diff
@@ -0,0 +1,1095 @@
+#------------------------------------------------------------------------------
+# Single Instance master threshold alarms
+resource "aws_cloudwatch_metric_alarm" "master_cpu_utilization_high" {
+  count                     = var.create_master_cpu_utilization_high_alarm ? 1 : 0
+  alarm_name                = "${var.db_instance_id}_cpu_utilization_high_static"
+  comparison_operator       = "GreaterThanThreshold"
+  evaluation_periods        = local.thresholds["CPUUtilizationEvaluationPeriods"]
+  metric_name               = "CPUUtilization"
+  namespace                 = "AWS/RDS"
```

Comment:
you can create a namespace local varible.
something like:
```
locals{
  cloudwatch_namespace = "AWS/RDS"
}
```

Reply:
done